media-sound/musepack-tools/files/musepack-tools-465-musl.patch

```diff
@@ -0,0 +1,24 @@
+diff -uNr a/mpcenc/keyboard.c b/mpcenc/keyboard.c
```

Comment:
- Please run scrub-patch from iwdevtools
- Please send upstream if possible and include the link at the top of the patch
- Please include links to any relevant Gentoo bugs at the top of the patch